remoting/subsystem/src/main/resources/org/jboss/as/remoting/LocalDescriptions.properties

```diff
@@ -112,7 +112,7 @@ remote-outbound-connection.authentication-context=Reference to the authenticatio
 remote-outbound-connection.property=The XNIO Options that will be used during the connection creation.
 
 
-remoting.local-outbound-connection=Remoting outbound connections for local:// URI scheme.
+remoting.local-outbound-connection=Remotely establishing outbound connections for the local://URI scheme.The remoting connectors.
```

Comment:
Similar issue as my comment above. Also, the space between local:// and URI should be restored. The string 'URI' is not part of the URI, it's a separate word.